ui/app/pages/send/send-content/send-asset-row/send-asset-row.component.js

```diff
@@ -147,6 +150,9 @@ export default class SendAssetRow extends Component {
             />
           </div>
         </div>
+        { !insideDropdown && this.props.tokens.length > 0 && (
```

Comment:
Nit: Part of this condition is redundant; `this.props.tokens` must necessarily have a length greater than zero given that a token is being passed in here as the first parameter.
```suggestion
        { !insideDropdown && && (
```

Reply:
Good catch. Made the change @Gudahtt